snowshu/adapters/target_adapters/base_target_adapter.py

```diff
@@ -198,6 +201,10 @@ def _get_data_type(self, source_type: str) -> DataType:
                 '%s adapter does not support data type %s.', self.CLASSNAME, source_type)
             raise err
 
+    def quoted_dot_notation(self, rel: Relation) -> str:
```

Comment:
Since the `quoted_dot_notation` function is the same everywhere, we can probably put this back onto the `BaseSQLAdapater` base class instead of each individual class.